docs/src/main/asciidoc/_admin-guide/subsystem-configuration/Elytron_OIDC_Client.adoc

```diff
@@ -141,6 +143,23 @@ configuration). If the subsystem detects that the `OIDC` mechanism is indeed req
 activate the authentication mechanism automatically. Otherwise, no activation will occur and deployment
 will continue normally.
 
+=== Scope Configuration
+
+OpenID Connect allows the client to use scope values to request access to specific privileges for access tokens. An access token's scopes determine what resources will be available when using the access token to access an OAuth2 protected endpoint. The OpenID Connect https://openid.net/specs/openid-connect-core-1_0.html#ScopeClaims[specification] defines a few scope values including:
+
+* `profile` : requests access to the End-User's default profile Claims, such as name, family_name, given_name, middle_name, nickname, preferred_username, profile, picture, website, gender, birthday etc. 
+
+* `email` : requests access to the email and email_verified Claims.
+
+* `address` :  requests access to the address Claim. 
+
+* `phone` : requests access to the phone_number and phone_number_verified Claims.
+
+It is possible to configure the scope values that would be used in the OIDC authentication request using the `scope` attribute of a `secure-deployment` or a `secure-server` resource in the `elytron-oidc-client` subsystem.
```

Comment:
Thanks for the updates! Just to make this easier to spot for a reader who's skimming, just wondering if we could add `==== Subsystem Configuration` and `==== Deployment Configuration` headings to separate this sentence and the next one?